.jscsrc

```diff
@@ -6,6 +6,7 @@
     "requireLineFeedAtFileEnd": null,
     "requireTrailingComma": null,
     "requirePaddingNewLinesBeforeLineComments": null,
+    "validateLineBreaks": null,
```

Comment:
This rule validates `LF` vs `CRLF`.  Since we don't specify default line-endings via a `.gitattributes` file, this rule will fail when someone on Windows attempts to contribute code.